webapp/packages/plugin-data-import/src/DataImportDialog/DataImportFileUploader.m.css

```diff
@@ -0,0 +1,24 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+.header {
+  composes: theme-border-color-background from global;
+  overflow: hidden;
+  position: sticky;
+  top: 0;
+  z-index: 0;
+  border-bottom: 1px solid;
+}
+
+.table {
+  composes: theme-background-surface theme-text-on-surface from global;
+}
```

Comment:
are you sure that you need this style (especially background) because `<Group>` will set same styles